src/Native/Unix/System.Security.Cryptography.Native/pal_ssl.cpp

```diff
@@ -94,6 +94,13 @@ extern "C" SSL* CryptoNative_SslCreate(SSL_CTX* ctx)
 
 extern "C" int32_t CryptoNative_SslGetError(SSL* ssl, int32_t ret)
 {
+    // This pops off "old" errors left by other operations
+    // until the first and last error are the same
+    // this should be looked at again when OpenSsl 1.1 is migrated to
+    while(ERR_peek_error() != ERR_peek_last_error())
```

Comment:
Please run `clang-format -i` on this file.  (At least accepting any changes in this function)